### DISCUSSION TITLE
New communication protocol

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#442] [Q&A] [answered]
_Original report by Dessislav Gouzgounov:_ https://bugs.launchpad.net/stellarium/+bug/1669889

Hello,

I have built a new Open Source Telescope control system based on Arduino Due. The name of the project is rDUINOScope and more information can be found on the project website: http://rduinoscope.co.nf/

EDIT GZ 2018-12-24: Moved to http://rduinoscope.byethost24.com/?i=1 

The rDUINOScope is capable of receiving commands via Bluetooth - SlewTo, Status and others.
I'm now looking for a way to connect it to Stellarium and found your Plug-In which can handle Bluetooth communication.

I'd like to know the communication protocol of the Plug-In in order to implement it in rDUINOScope, or it could be that I send you my protocol and you include it in your plugin.

Please let me know :)

Best Regards,
Dessislav Gouzgounov

**Comment by gzotti** [comment, marked as answer]
See original discussion. We gave all help possible.
